config/locales/en.yml

```diff
@@ -32,7 +32,7 @@
 en:
   # start documentation links
   docs:
-    base_url: 'https://access.redhat.com/documentation/en-us/%{major_and_minor_version_path}/html'
+    base_url: 'https://access.redhat.com/documentation/en-us/%{docs_version}/html'
```

Comment:
The value of this variable is only for docs, and includes the product name and the version, so I found the old variable name quite confusing.